### PR TITLE
Feat/write transport tests

### PR DESCRIPTION
**Pull request by Alex-A4**
- Fix creating transport objects
- Add gql and jrpc transport tests in dart side

### CODE REVIEW

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  const name = 'Mainnet (GQL)';
+  const networkId = 1;
+  const networkGroup = 'mainnet';
+  const endpoint =
+      'https://mainnet.evercloud.dev/89a3b8f46a484f2ea3bdd364ddaee3a3/graphql';
+
+  const kDefaultLatencyDetectionInterval = 60000;
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:36]: Const prefixes :)

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+  const kDefaultMaxLatency = 60000;
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:38]: Const prefixes :)

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+  const kDefaultEndpointSelectionRetryCount = 5;
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:40]: Const prefixes :)

File: packages/flutter_nekoton_bridge/example/pubspec.yaml [hunk 4]
```diff
@@ -44,6 +44,8 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
   freezed_annotation: ^2.2.0
+  #  Used for tests
+  http: 0.13.5
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/example/pubspec.yaml:48]: Only for tests? Then in should be in dev_dependencies imho.

**Review by nesquikm — CHANGES_REQUESTED** [review]
This is not an integration tests, its frickin e2e! :)

**Comment by Alex-A4** [comment]
> This is not an integration tests, its frickin e2e! :)

It is, but we can't write such tests with rust functionality without running it on device. Or we need to solve macos running (if it would help)

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart (hunk 1 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:36]: -

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart (hunk 2 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:38]: -

File: packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart (hunk 3 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/example/integration_test/transport/gql_transport_test.dart:40]: -

File: packages/flutter_nekoton_bridge/example/pubspec.yaml (hunk 4 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/example/pubspec.yaml:48]: -

**Pull request merged**